runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/hotcold/HotColdLocation.java

```diff
@@ -176,12 +182,159 @@
+		KANDARIN_SINCLAR_MANSION,
+		KANDARIN_CATHERBY,
+		KANDARIN_GRAND_TREE,
+		KANDARIN_SEERS,
+		KANDARIN_MCGRUBORS_WOOD,
+		KANDARIN_FISHING_BUILD,
+		KANDARIN_WITCHHAVEN,
+		KANDARIN_NECRO_TOWER,
+		KANDARIN_FIGHT_ARENA,
+		KANDARIN_TREE_GNOME_VILLAGE,
+		KANDARIN_GRAVE_OF_SCORPIUS,
+		KANDARIN_KHAZARD_BATTLEFIELD,
+		KANDARIN_WEST_ARDY,
+		KANDARIN_SW_TREE_GNOME_STRONGHOLD,
+		KANDARIN_OUTPOST,
+		KANDARIN_BAXTORIAN_FALLS,
+		KANDARIN_BA_AGILITY_COURSE,
+		KARAMJA_MUSA_POINT,
+		KARAMJA_BRIMHAVEN_FRUIT_TREE,
+		KARAMJA_WEST_BRIMHAVEN,
+		KARAMJA_GLIDER,
+		KARAMJA_KHARAZI_NE,
+		KARAMJA_KHARAZI_SW,
+		KARAMJA_CRASH_ISLAND,
+		MISTHALIN_VARROCK_STONE_CIRCLE,
+		MISTHALIN_LUMBRIDGE,
+		MISTHALIN_LUMBRIDGE_2,
+		MISTHALIN_GERTUDES,
+		MISTHALIN_DRAYNOR_BANK,
+		MISTHALIN_LUMBER_YARD,
+		MORYTANIA_BURGH_DE_ROTT,
+		MORYTANIA_PORT_PHASMATYS,
+		MORYTANIA_HOLLOWS,
+		MORYTANIA_SWAMP,
+		MORYTANIA_HAUNTED_MINE,
+		MORYTANIA_MAUSOLEUM,
+		MORYTANIA_MOS_LES_HARMLESS,
+		MORYTANIA_MOS_LES_HARMLESS_BAR,
+		MORYTANIA_DRAGONTOOTH_NORTH,
+		MORYTANIA_DRAGONTOOTH_SOUTH,
+		WESTERN_PROVINCE_EAGLES_PEAK,
+		WESTERN_PROVINCE_PISCATORIS,
+		WESTERN_PROVINCE_PISCATORIS_HUNTER_AREA,
+		WESTERN_PROVINCE_ARANDAR,
+		WESTERN_PROVINCE_ELF_CAMP_EAST,
+		WESTERN_PROVINCE_ELF_CAMP_NW,
+		WESTERN_PROVINCE_LLETYA,
+		WESTERN_PROVINCE_TYRAS,
+		WESTERN_PROVINCE_ZULANDRA,
+		WILDERNESS_5,
+		WILDERNESS_12,
+		WILDERNESS_20,
+		WILDERNESS_27,
+		WILDERNESS_28,
+		WILDERNESS_32,
+		WILDERNESS_35,
+		WILDERNESS_37,
+		WILDERNESS_38,
+		WILDERNESS_49,
+		WILDERNESS_54,
+		ZEAH_BLASTMINE_BANK,
+		ZEAH_BLASTMINE_NORTH,
+		ZEAH_LOVAKITE_FURNACE,
+		ZEAH_LOVAKENGJ_MINE,
+		ZEAH_SULPHR_MINE,
+		ZEAH_SHAYZIEN_BANK,
+		ZEAH_OVERPASS,
+		ZEAH_LIZARDMAN,
+		ZEAH_COMBAT_RING,
+		ZEAH_SHAYZIEN_BANK_2,
+		ZEAH_LIBRARY,
+		ZEAH_HOUSECHURCH,
+		ZEAH_DARK_ALTAR,
+		ZEAH_ARCEUUS_HOUSE,
+		ZEAH_ESSENCE_MINE,
+		ZEAH_ESSENCE_MINE_NE,
+		ZEAH_PISCARILUS_MINE,
+		ZEAH_GOLDEN_FIELD_TAVERN,
+		ZEAH_MESS_HALL,
+		ZEAH_WATSONS_HOUSE,
+		ZEAH_VANNAHS_FARM_STORE,
+		ZEAH_FARMING_GUILD_W,
+		ZEAH_DAIRY_COW,
+		ZEAH_CRIMSON_SWIFTS
+	);
+
 	private final WorldPoint worldPoint;
 	private final HotColdArea hotColdArea;
-	private final String area;
+	private final String areaDescription;
+	private final int digRadius;
 
-	public Rectangle2D getRect()
+	HotColdLocation(WorldPoint worldPoint, HotColdArea hotColdArea, String areaDescription)
 	{
-		return new Rectangle(worldPoint.getX() - 4, worldPoint.getY() - 4, 9, 9);
+		this(worldPoint, hotColdArea, areaDescription, HotColdTemperature.MASTER_VISIBLY_SHAKING.getMaxDistance());
+	}
+
+	/**
+	 * @return A {@link Rectangle} defining the area which the solution can be solved at. That is, an area which the
+	 *         player should dig, activate a location device, or perform a specific action. This will usually correspond
+	 *         with {@link HotColdTemperature#getMaxDistance()} for the temperature value which is nearest the solution
```

Comment:
Java 11 apparently doesn't like the `{@link}` to a lombok-generated method, so I'll need to update this to get that build passing.